DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Plugin/CefSharpPanel.xaml

```diff
@@ -11,7 +11,7 @@
     <cefSharp:ChromiumWebBrowser
       Name="Browser"
       Grid.Row="0"
-      Address="https://deploy-preview-2076--boisterous-douhua-e3cefb.netlify.app/" />
+      Address="https://dui3--boisterous-douhua-e3cefb.netlify.app/" />
```

Comment:
How can we unify this on every connector (regardless of WebView2,Cef...? This seems like point of failure in the future

Reply:
@oguzhankoral has added this as a static for Revit or for the webview or possible both